formlibrary/models/inventory.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from .service import Service
+from utils.models import CreatedModifiedBy, CreatedModifiedDates
+
+
+class Inventory(Service, CreatedModifiedBy, CreatedModifiedDates):
+    serviceid = models.ForeignKey(Service, null=True, on_delete=models.SET_NULL)
```

Comment:
@epanjwani 
We don't need an explicit `service_id` attribute, since Inventory inherits from Service.
The issues describing the fields, although slightly technical, are more of business requirements, and a few implementation details have been discussed before (inheriting from Service in this case)